### ISSUE TITLE
Fluent-bit 1.5.7  - Forward processor error - invalid 'compressed' option

### ISSUE THREAD

**Issue by hafkaM** [fluent/fluent-bit#2644]
## Bug Report

**Describe the bug**
Fluent-bit is receive a errors from forward processor - [error] [input:forward:forward.0] invalid 'compressed' option

**To Reproduce**
From version 1.5. (docker fluent/fluent-bit:1.5.7)
The error occurs whenever:
Always if I send messages from fluentd to fluentbit using a forward plugin. I get this error. 
- Example log message if applicable:
```
[2020/10/06 04:31:02] [error] [input:forward:forward.0] invalid 'compressed' option
```
- Steps to reproduce the problem:

**Expected behavior**
Forwarding between fluentd and fluentbit

**Your Environment**

* Version used: 1.5.7 - docker installation
* Configuration:

Fluentd part of the config:
```
<match o2tv.events.raw>
  @type forward
  send_timeout 15s
  recover_wait  5s
  hard_timeout 15s

  <server>
    name local_5148
    host xx.xx.xx.xx
    port 5148
    weight 60
  </server>

  <secondary>
    @type file
    path /fluentd/log/td-agent/forward-failed
  </secondary>
</match>
```
Fluent-bit part of the config
```
[INPUT]
    Name          forward
    Host          0.0.0.0
    Port          5148
    Match         *
    Buffer_Chunk_Size 64MB
    Buffer_Max_Size 128MB
    Chunk_Size  64MB
    Buffer_Size 64MB
```

* Operating System and version: Centos 7

**Comment by edsiper** [comment]
Thanks for reporting this. I've pushed the fix bb0c95b4 . 

A workaround for the moment before the next release (Thursday) will be to force the gzip compression mode.

**Comment by hafkaM** [comment]
Thanks a lot for the fix in the new version 1.6. 
Is support for gzip compression or other compression planned in the next version?
